binstar_client/utils/__init__.py

```diff
@@ -247,8 +245,8 @@ def recursive_update(d, u):
             d[k] = u[k]
     return d
 
-DEFAULT_URL = 'https://api.anaconda.org'
```

Comment:
@malev I don't think this will work as expected. I think that the environment variables should have a defined precedence that is stronger than the vars in the config file. This will not use the environment variable when the config variable url is set [because it is used here](https://github.com/Anaconda-Server/anaconda-client/blob/api-on-premises/binstar_client/utils/__init__.py#L161)